utils/connectApp.js

```diff
@@ -74,12 +74,34 @@ const connectApp = async (req, res) => {
       return res.status(200).json({data: shaResult, code: 200});
     }
 
+    else if (api === 'getQuestSurveyFromGitHub') {
+      if (req.method !== 'GET') {
+        return res.status(405).json(getResponseJSON('Only GET requests are accepted!', 405));
+      }
+
+      if (!req.query.sha) {
+        return res.status(400).json(getResponseJSON('Sha parameter is required!', 400));
+      }
+
+      if (!req.query.path) {
+        return res.status(400).json(getResponseJSON('Path parameter is required!', 400));
+      }
+
+      const sha = req.query.sha;
```

Comment:
Can we use `const {sha, path} = req.query;`?

Reply:
sure, updated to use destructuring